exercises/react/src/react.h

```diff
@@ -0,0 +1,29 @@
+#ifndef REACT_H
+#define REACT_H
+
+struct reactor;
+struct cell;
+
+typedef int (*compute1) (int);
+typedef int (*compute2) (int, int);
+
+struct reactor *create_reactor();
+// destroy_reactor should free all cells created under that reactor.
```

Comment:
Well, to me there are two separate levels of abstraction: malloc/free at the implementation level and create/destroy at the interface level (which elides certain details about what's actually happening). This is an odd case where we're at the boundary of those two concerns. However, since these are instructions for the implementer, I'm okay with it still saying "free".